packages/tools/devtools/devtools-browser-extension/e2e-tests/chromebrowserdevtool.test.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { globals } from "../jest.config";
+import { retryWithEventualValue } from "@fluidframework/test-utils";
+// import { exec } from "child_process";
+
+describe("chrome browser...", () => {
+	const getValue = async (index: number, expectedValue: string) =>
+		retryWithEventualValue(
+			() =>
+				page.evaluate((i: number) => {
+					const divs = document.getElementsByClassName("text-area");
+					const textAreaElements = divs[i].getElementsByTagName("textarea");
+					const textarea = textAreaElements[0] as HTMLTextAreaElement;
+					if (textarea) {
+						return textarea.value;
+					}
+					return "-----undefined-----";
+				}, index),
+			(actualValue) => actualValue === expectedValue,
+			"not propagated" /* defaultValue */,
+		);
+
+	beforeAll(async () => {
+		// Wait for the page to load first before running any tests
+		// so this time isn't attributed to the first test
+		await page.goto(globals.PATH, { waitUntil: "load", timeout: 0 });
+	}, 10000);
+
+	beforeEach(async () => {
+		await page.goto(globals.PATH, { waitUntil: "load" });
+		await page.waitForSelector(".text-area");
```

Comment:
This is waiting for the collaborative text area control to show up, right? Can we also verify that the devtools UI is up?

Reply:
Adding

Reply:
Tong and I chatted about this offline. We're going to punt on this for now. Puppeteer is going to require us to launch the devtools extension in a separate page. Follow up work will start by adding a smoke test for launching the extension UI by itself, and follow up tests will launch both pages side by side to test interactions.